jwebunit-webdriver-plugin/pom.xml

```diff
@@ -9,17 +29,7 @@
     <artifactId>jwebunit-webdriver-plugin</artifactId>
     <name>Webdriver Plugin</name>
     <description>Webdriver plugin for JWebUnit.</description>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <testFailureIgnore>true</testFailureIgnore>
```

Comment:
Yep, I was just poking this part to see if the test failures it was ignoring had been fixed in the meantime. Looks like that's not the case:

```
Results :

Failed tests: 
  FormAssertionsTest.testSetLabels:353 unexpected value of field for label [label1] expected:<[]two> but was:<[one]two>
  IElementTest.testAttributeJavascript:229 expected:<[]Tue Jan 10 07:47:22 ...> but was:<[initial]Tue Jan 10 07:47:22 ...>
  TestContextTest.testClientSideUserAgentOverride:86 Expected text not found in current page: [Browser user-agent: Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.8.0.3) Gecko/20060426 Firefox/1.5.0.3]
 Page content was: [<?xml version="1.0" encoding="ISO-8859-1"?>
<html>
  <head/>
  <body>
    <script language="javascript">
//<![CDATA[

			document.write("Browser code name: " +navigator.appCodeName);
			document.write("Browser name: " +navigator.appName);
			document.write("Browser version: "+navigator.appVersion);
			document.write("Browser user-agent: "+navigator.userAgent);
		
//]]>
    </script>
    Browser code name: MozillaBrowser name: NetscapeBrowser version: 5.0 (Windows)Browser user-agent: Mozilla/5.0 (Windows NT 6.1; rv:38.0) Gecko/20100101 Firefox/38.0
	
  </body>
</html>
]
  WebCookieTest.testAddAnotherCookie:61 Expected text not found in current page: [cookie1=Cookievalue1]
 Page content was: [Submitted cookies
Cookies are:
return]
  WebCookieTest.testAddCookie:54 Expected text not found in current page: [cookie1=Cookievalue1]
 Page content was: [Submitted cookies
Cookies are:
return]
  WebCookieTest.testCookieMatchLastCookie:91 Could not find Cookie with name [JSESSIONID]
Tests in error: 
  IElementTest.testComments:240 » UnsupportedOperation Not supported yet.
  JavaScriptTest.testAlert:52 » UnsupportedOperation Not supported yet.
  JavaScriptTest.testConfirm:72 » UnsupportedOperation Not supported yet.
  JavaScriptTest.testInvalidAlertOnPageLoad:57 » UnsupportedOperation Not suppor...
  JavaScriptTest.testMultipleAlerts:67 » UnsupportedOperation Not supported yet.
  JavaScriptTest.testPrompt:79 » UnsupportedOperation Not supported yet.
  JavaScriptTest.testPromptCanceled:85 » UnsupportedOperation Not supported yet.
  NonHtmlContentTest.testImageContent:54 » Runtime Error when writing to file
  ResponseServletTest.testTimeout:114 » UnsupportedOperation Not supported yet.

Tests run: 257, Failures: 6, Errors: 9, Skipped: 0
```

This is at least a better outcome than failing in JWebUnitTest to start jetty (which meant nothing was working with the jwebunit-webdriver-plugin tests).

I'll revert this ignore for now, but it would be better to find a more targeted way to ignore UnsupportedOperationExceptions, so we can detect other regressions in how well this part works. Maybe through a Rule in the affected tests?